service/service.hpp

```diff
@@ -21,24 +21,21 @@ namespace raven
   class service
   {
   public:
-    service(std::filesystem::path db_path = std::filesystem::current_path() / "albinos_service.db") noexcept : db_{
-        db_path}
+    explicit service(std::filesystem::path db_path = std::filesystem::current_path() / "albinos_service.db") noexcept
+    : db_{std::move(db_path)}
     {
         VLOG_SCOPE_F(loguru::Verbosity_INFO, "service constructor");
-        DVLOG_F(loguru::Verbosity_INFO, "register error_event libuv listener: %s",
-                "<uvw::ErrorEvent>([this](auto const &error_event, auto &)");
+        DVLOG_F(loguru::Verbosity_INFO, "register error_event libuv listener");
```

Comment:
"register" -> "registering" when you're going to and "registered" when you just have. I know this wasn't introduced by you but seems like a good occasion to fix.

Reply:
(I'm not gonna repeat myself for all the other cases)